src/main/java/org/owasp/esapi/logging/java/ESAPICustomJavaLevel.java

```diff
@@ -18,7 +18,7 @@
 import java.util.logging.Level;
 
 /**
- *  A custom logging level defined between Level.SEVERE and Level.WARNING in logger.
+ *  Definitions of customized Java Logging Level options to map ESAPI behavior to the desired Java Log output behaviors.
```

Comment:
Probably still advisable to leave this comment here even if the class name is not changed.

Reply:
This was moved to the ERROR_LEVEL entry, since it is specific to that instance. (line 27-31 of this file)